server/middleware/checkUserId.js

```diff
@@ -0,0 +1,14 @@
+import validation from './validation';
+import messages from '../utils/messages';
+
+module.exports = (req, res, next) => {
+  // Check if userid is an integer
+  const { error } = validation.validateUserId(req.params);
+  if (error) {
+    return res.status(400).json({
+      status: 400,
+      error: messages.invalidUserId,
+    });
```

Comment:
you already have a function for this, can you use it??

Reply:
Sure.